.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -14,7 +14,7 @@ assignees: ''
 <!--A clear and concise description of what the bug is.-->
 
 **To Reproduce**
-Steps to reproduce the behavior:
+<!--Steps to reproduce the behavior:-->
```

Comment:
All the steps here should be commented. See this as reference https://raw.githubusercontent.com/ChainSafe/lodestar/master/.github/ISSUE_TEMPLATE/bug_report.md